packages/flutter/test/rendering/debug_test.dart

```diff
@@ -231,7 +231,8 @@ void main() {
       rootLayer,
       const Rect.fromLTWH(0, 0, 500, 500),
     );
-    root.paint(context, const Offset(40, 40));
+    context.paintChild(root, const Offset(40, 40));
```

Comment:
minor change: since the RenderOpacity would be a repaint boudary in this case, it would have gone down context.paintChild path to get to compositing correctly